DOL.WHD.Section14c.Web/src/modules/components/resultsTable/resultsTableController.js

```diff
@@ -31,8 +31,12 @@ module.exports = function(ngModule) {
             }
         },
         buttons: [
-          'copy','csv'
+          { extend: 'copy', text: 'Copy<span class="hide"> table data to clipboard</span>' },
+          { extend: 'csv', text: '<span class="hide">Generate a </span>CSV<span class="hide"> file from the table data</span>' }
          ],
+        "language": {
+          "search": "Search<span class='hide'> Table Data</span>"
```

Comment:
For consistency, lines 37 and 38 should use single quotes.  The HTML attribute value should be in double quotes.  E.g., `'Search<span class="hide"> Table Data</span>'`